requestor.js

```diff
@@ -30,9 +30,7 @@ function Requestor(api_key, config) {
   });
 
   function resource_path(latest) {
-    return latest ? '/api/eval/latest-features' : '/api/eval/features';
-   // TODO Update to the following v2 paths: 
-   // return latest ? '/sdk/latest-flags': '/sdk/flags';
+    return latest ? '/sdk/latest-flags': '/sdk/flags';
```

Comment:
Let's go ahead and just always return '/sdk/latest-flags' here since we no longer use the non-latest endpoint.